README.md

```diff
@@ -109,3 +109,9 @@ The tibble print method draws inspiration from
 [r-frame](https://github.com/patperry/r-frame). Like `data.table()`,
 `tibble()` doesn’t coerce strings to factors by default, doesn’t change
 column names, and doesn’t use rownames.
+
```

Comment:
Do we add a horizontal row (three dashes) here?

Reply:
I think it's automatically added because of the header below, now.

Reply:
I see it. We might want to add an extra separator above the caption, but I don't have a strong opinion.